src/cryptonote_core/blockchain.cpp

```diff
@@ -3724,7 +3726,7 @@ bool Blockchain::prepare_handle_incoming_blocks(const std::list<block_complete_e
       m_blocks_longhash_table.clear();
       for (uint64_t i = 0; i < threads; i++)
       {
-        thread_list.push_back(new boost::thread(&Blockchain::block_longhash_worker, this, height + (i * batches), std::cref(blocks[i]), std::ref(maps[i])));
+        thread_list.push_back(new boost::thread(attrs, boost::bind(&Blockchain::block_longhash_worker, this, height + (i * batches), std::cref(blocks[i]), std::ref(maps[i]))));
```

Comment:
What gcc version was used? I'm curious because of your IRC message about needing `boost::bind`. Looks like C++11 variadic arguments are disabled for Boost.Thread in this environment, and I wanted to poke around to what limitations this environment has.